internal/controllers/import_controller.go

```diff
@@ -250,8 +251,8 @@ func (r *CAPIImportReconciler) reconcileNormal(ctx context.Context, capiCluster
 
 	log.Info("found cluster name", "name", rancherCluster.Status.ClusterName)
 
-	if rancherCluster.Status.AgentDeployed {
-		log.Info("agent already deployed, no action needed")
+	if conditions.IsTrue(rancherCluster, managementv3.ClusterConditionReady) {
```

Comment:
does this line break V1 tests?

Reply:
Seems it was not needed, so reverted it back. It should not on it’s own, but the problem was the cleanup.